internal/release/main.go

```diff
@@ -96,6 +96,7 @@ func generateFile(name, version string) error {
 			*newPlatform(version, "x86_64", "linux", "Debian 9 / Ubuntu 16.04 + 18.04", "deb"),
 			*newPlatform(version, "x86_64", "linux", "Red Hat + CentOS 6, 7, 8 / SUSE 12 + 15 / Amazon Linux", "rpm"),
 			*newPlatform(version, "x86_64", "windows", "Microsoft Windows", "zip"),
+			*newPlatform(version, "x86_64", "windows", "Microsoft Windows", "msi"),
 			*newPlatform(version, "x86_64", "macos", "macOS", "zip"),
```

Comment:
also, should we add the arm here? 🤔

Reply:
Should the arm zip be already available at `https://fastdl.mongodb.org/mongocli/mongocli_1.14.0_macos_arm64.zip `?

Reply:
no,we have not relased since we added arm so it won't be there until our next release

Reply:
same for all these changes, they won't go live until we release